public/app/features/dashboard/components/ShareModal/ShareLink.tsx

```diff
@@ -99,7 +94,7 @@ export class ShareLink extends PureComponent<Props, State> {
         <FieldSet>
           <Field
             label="Lock time range"
-            description={isRelativeTime ? 'Transforms the current relative time range to an absolute time range' : ''}
+            description={isRelativeTime && 'Transforms the current relative time range to an absolute time range'}
```

Comment:
Could this be rendered `"null"` or `"false"` strings if `isRelativeTime` is a falsy value?

Reply:
this actually shouldn't be here at all!!!!!! unrelated change i was testing out (to answer your exact question 😅